### PR TITLE
Vs feature stores brands screen

### PR DESCRIPTION
**Pull request by verasunyy**
link fave context to UI:
-add fave context to storesbrands screen
-add fave context to single store and single brand screen
-add query to stroesbrands screen

### CODE REVIEW

File: client/js/components/StoreCard/StoreCard.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import FaveStoresContext from '../../context/FaveStoresContext';
+
+const StoreCard = ({
+  store, navigation
+}) => (
+    <TouchableOpacity
+      onPress={() => navigation.navigate('store', { store })}
+    >
+      <View>
+        {/* <Text>{store.storeLogo}</Text> */}
```
Comment (siddharthparmar7) [inline client/js/components/StoreCard/StoreCard.js]: remove commented code.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
This pull request has the same files as your other branch. Please push the correct files and remove merge conflicts. Thank you.

File: client/js/components/StoreCard/StoreCard.js [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
+            //   source={{uri: `${store.images}`}}
+            source={{ uri: 'https://via.placeholder.com/150' }}
+            style={styles.image}
+            resizeMode="contain"
+          />
+        </View>
+        <View style={styles.storeContentContainer}>
+          {/* <Text>{store.title}</Text>
         <Text>{store.address}</Text>
         <Text>{store.phone}</Text> */}
```
Comment (siddharthparmar7) [inline client/js/components/StoreCard/StoreCard.js]: remove these comments.

File: client/js/screens/StoresBrands/StoresBrands.js [hunk 3]
```diff
@@ -1,12 +1,43 @@
...
+import React, { Component } from 'react';
+import { View, Text, FlatList, TouchableOpacity } from 'react-native';
+import StoreCard from '../../components/StoreCard'
+class StoresBrands extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      displayStore: true
+    }
+  }
```
Comment (siddharthparmar7) [inline client/js/screens/StoresBrands/StoresBrands.js]: declare your state in your stateful component. Container file.

File: client/js/screens/StoresBrands/StoresBrandsContainer.js [hunk 4]
```diff
@@ -1,13 +1,76 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import StoresBrands from './StoresBrands';
-import {withNavigation} from 'react-navigation';
+// import { withNavigation } from 'react-navigation';
```
Comment (siddharthparmar7) [inline client/js/screens/StoresBrands/StoresBrandsContainer.js]: remove commented code.

File: client/react-native.config.js [hunk 5]
```diff
@@ -1,3 +1,3 @@
-module.exports = {
-  assets: ['./js/assets/fonts'],
-};
+// module.exports = {
+//   assets: ['./js/assets/fonts'],
+// };
```
Comment (siddharthparmar7) [inline client/react-native.config.js]: are you removing fonts?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes. Thank you.

File: client/react-native.config.js (hunk 5 above)
Comment (verasunyy) [reply client/react-native.config.js]: waiting on link

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please resolve your merge conflicts. Thank you.

File: client/js/screens/Profile/Profile.js [hunk 6]
```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import { View, Text, Button } from 'react-native';
 import { createToken, deleteToken, queryToken } from "../../config/models/authentication"
 import { withNavigation } from 'react-navigation';
-const Profile = ({ navigation }) => {
+import AsyncStorage from '@react-native-community/async-storage';
+const Profile = ({ navigation, user }) => {
+  console.log(user)
```
Comment (siddharthparmar7) [inline client/js/screens/Profile/Profile.js:7]: remove console log.

File: client/js/screens/Profile/ProfileContainer.js [hunk 7]
```diff
@@ -1,13 +1,50 @@
...
+    this.state = {
+      userId: ""
+    }
+  }
   static navigationOptions = {
     title: 'My Profile',
   };
+  getUser = async () => {
+    const userToken = await queryToken('userToken');
+    console.log(userToken.userId)
```
Comment (siddharthparmar7) [inline client/js/screens/Profile/ProfileContainer.js:31]: remove console log.

File: client/js/config/styles.js [hunk 8]
```diff
@@ -1,6 +1,6 @@
-export const fonts = {
-  baseFont: 'Lato-Regular',
-};
+// export const fonts = {
+//   baseFont: 'Lato-Regular',
+// };
```
Comment (siddharthparmar7) [inline client/js/config/styles.js:3]: remove comments.

**Review by siddharthparmar7 — APPROVED** [review]
Please make changes in your next pull request. Thank you.

**Pull request merged**